File: src/main/java/com/extendedae_plus/mixin/bridge/HelperAssemblerMatrixModifier.java
package com.extendedae_plus.mixin.bridge;

import com.glodblock.github.extendedae.common.tileentities.matrix.TileAssemblerMatrixCrafter;

public interface HelperAssemblerMatrixModifier {
    void eaep$addCrafter(TileAssemblerMatrixCrafter crafter);

    void eaep$addSpeedCore();

    void eaep$updateCrafter(TileAssemblerMatrixCrafter crafter);

    void eaep$markUploadCore();

    boolean eaep$hasUploadCore();
}

File: src/main/kotlin/com/extendedae_plus/mixin/helper/HelperAssemblerMatrixModifier.kt
package com.extendedae_plus.mixin.helper

import com.glodblock.github.extendedae.common.tileentities.matrix.TileAssemblerMatrixCrafter

interface HelperAssemblerMatrixModifier {
    fun `eaep$addCrafter`(crafter: TileAssemblerMatrixCrafter)

    fun `eaep$addSpeedCore`()

    fun `eaep$updateCrafter`(crafter: TileAssemblerMatrixCrafter)

    fun `eaep$markUploadCore`()

    fun `eaep$hasUploadCore`(): Boolean
}
